client/components/tile-grid/docs/example.jsx

```diff
@@ -0,0 +1,55 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Tile from 'components/tile-grid/tile';
+import TileGrid from 'components/tile-grid';
+
+/**
+ * Main
+ */
+class TileGridExample extends PureComponent {
+	static displayName = 'TileGrid';
+
+	render() {
+		return (
+			<TileGrid>
+				<Tile
+					buttonLabel={ 'Start with a blog' }
+					description={ 'To share your ideas, stories, and photographs with your followers.' }
+					image={ '/calypso/images/illustrations/type-blog.svg' }
+					onClick={ noop }
```

Comment:
Hmm, would be nice if we didn't have to specify that, i.e. make it a `defaultProp`. (Or maybe better yet, make the default prop `null` and only invoke if it isn't.)

Reply:
Oh, for the case of devdocs specifically, I think we sometimes choose to call `console.log()` to provide some meaningful feedback and demonstrate how props like `onClick` are used.

Reply:
Yeah, makes sense. 

Well, we don't need to specify `onClick` at all - it can remain `undefined` if it's undefined, which kinda makes sense to me. And for demonstration purposes, I've used a `console.log` in one of the tiles.